resources/views/components/layout.blade.php

```diff
@@ -15,14 +15,20 @@
     <title>Elena Molano</title>
 </head>
 
-<body class="h-full">
+<body class="h-full relative">
     <div class="flex justify-center">
         <button id="back-to-top-btn" class="hidden fixed top-[40px] p-5" style="z-index: 5;">
             <i class="fa-solid fa-chevron-up text-[32px]" id="chevron-up-icon"></i>
         </button>
     </div>
 
     {{ $slot }}
+
+    @if (session('status'))
+    <div class="absolute bottom-0 right-0">
+        {{ session('status') }}
+    </div>
+    @endif
```

Comment:
Peut être faire un composant

Reply:
done with added animation and js to hide a flash-message